code/game/machinery/doors/door.dm

```diff
@@ -77,9 +77,14 @@
 		var/mob/living/M = AM
 		if(world.time - M.last_bumped <= 10) return	//Can bump-open one airlock per second. This is to prevent shock spam.
 		M.last_bumped = world.time
-		if(!M.restrained() && M.mob_size > MOB_SIZE_SMALL)
-			bumpopen(M)
-		return
+		if(!M.restrained())

```

Comment:
If not then they better now. I fully expect Ian to get arrested for eating all the chef's food. Also if they're going to have IDs might as well try it out